motionplan/constraint_test.go

```diff
@@ -20,7 +20,7 @@ import (
 func TestIKTolerances(t *testing.T) {
 	logger := golog.NewTestLogger(t)
 
-	m, err := frame.ParseModelJSONFile(utils.ResolveFile("referenceframe/testjson/varm.json"), "")
+	m, err := frame.ParseModelJSONFile(utils.ResolveFile("referenceframe/testjson/ur5eDH.json"), "")
```

Comment:
@raybjork, here's one place where I took out the varm as well. Planning changed from a 2dof -> 6dof system.